test/testmodels/customtypes/compositeprimarykey.yo.go

```diff
@@ -371,10 +371,10 @@ func ReadCompositePrimaryKeysByZError(ctx context.Context, db YORODB, keys spann
 	return res, nil
 }
 
-// FindCompositePrimaryKeysByZYError retrieves multiple rows from 'CompositePrimaryKeys' as a slice of CompositePrimaryKey.
+// FindCompositePrimaryKeysByYZError retrieves multiple rows from 'CompositePrimaryKeys' as a slice of CompositePrimaryKey.
 //
 // Generated from index 'CompositePrimaryKeysByError3'.
-func FindCompositePrimaryKeysByZYError(ctx context.Context, db YORODB, e int8) ([]*CompositePrimaryKey, error) {
```

Comment:
I ran it using Spanner emulator 1.5.15. I don't know the details, but the Storing column names returned by `INFORMATION_SCHEMA` are now sorted alphabetically. The name of this function has been changed due to this change.